src/crimson/os/seastore/linked_tree_node.h

```diff
@@ -217,7 +217,9 @@ class BaseChildNode {
 // indirect mapping.
 template <typename T, typename node_key_t>
 inline BaseChildNode<T, node_key_t>* get_reserved_ptr() {
-  return (BaseChildNode<T, node_key_t>*)0x1;
+  //TODO: using instant integers as invalid pointers may
+  //	  not be a good practice.
+  return (BaseChildNode<T, node_key_t>*)0x10;
```

Comment:
This possibly fixes the 8-byte aligned address issue -  https://tracker.ceph.com/issues/70202.
Although I'm not sure that this is the actual cause of the recent arm64 timeouts- https://tracker.ceph.com/issues/70254.

After verifying that this fixes 70202, we could merge this and still keep the timed out tests omitted for arm64.

Reply:
Yeah.